src/components/TestLevelSelector/TestLevelSelectorItem/TestLevelSelectorItem.js

```diff
@@ -3,22 +3,28 @@ import PropTypes from 'prop-types';
 import Button from '@material-ui/core/Button';
 import { Link } from 'react-router-dom';
 import './TestLevelSelectorItem.scss';
+import { startTest } from '../../../api/start-test';
 
 import { Trans } from '@lingui/macro';
 
-export const TestLevelsSelectorItem = (props) => {
+export const TestLevelsSelectorItem = ({titleEN, titleRU, descriptionEN, descriptionRU, level}) => {
 
   return (
     <div className='test-level-selector-item'>
-      <div className='title'><Trans>{props.titleEN}{props.titleRU}</Trans></div>
-      <div className='description'><Trans>{props.descriptionEN}{props.descriptionRU}</Trans></div>
+      <div className='title'>
+        {localStorage.getItem('language') === 'eng' ? titleEN : titleRU}
```

Comment:
You can use constants from src/constants/localStorageConstants.js instead of 'language'